library/core/src/convert/mod.rs

```diff
@@ -484,11 +466,8 @@ pub trait TryInto<T>: Sized {
 /// assert!(try_successful_smaller_number.is_ok());
 /// ```
 ///
-/// [`try_from`]: trait.TryFrom.html#tymethod.try_from
-/// [`TryInto`]: trait.TryInto.html
-/// [`i32::MAX`]: ../../std/i32/constant.MAX.html
+/// [`i32::MAX`]: crate::i32::MAX
```

Comment:
This links to the module constants, which are deprecated.
```suggestion
```
https://doc.rust-lang.org/nightly/std/i32/

Reply:
removing it in my local was breaking the doc build

Reply:
build [failed](https://github.com/rust-lang/rust/pull/75969/checks?check_run_id=1034624872#step:23:1630) after removing this 

```sh
413 | /// giving the [`i64`]'s value modulo [`i32::MAX`]) or by simply returning
    |                                        ^^^^^^^^^^ cannot be resolved, ignoring
```